src/index.js

```diff
@@ -336,7 +336,8 @@ function removeNodeFromHost(host, node) {
 }
 
 function removeSlotFromRoot(root, node) {
-  node.assignedNodes().forEach(slotNodeFromSlot);
+  const assignedNodes = Array.prototype.slice.call(node.assignedNodes());
```

Comment:
Interesting. Cool, then yeah, don't worry about it. But maybe make a note about possible performance hits as a comment or something? Maybe that's not necessary?